.github/workflows/register-buildpack.yml

```diff
@@ -0,0 +1,143 @@
+                process.env.NAMESPACE_REPO,
+                process.env.VERSION
+              )
+            } catch (error) {
+              console.error(error)
+              process.exit(1)
+            }
+            console.log(`::set-env name=REGISTRY_OWNERS::${registryOwners.replace(/\r?\n|\r/g, " ")}`)
+
+      - uses: actions/github-script@v2
+        name: Authenticate
+        with:
+          github-token: ${{ secrets.BOT_TOKEN }}
+          script: |
+            const path = require('path')
+            const scripts = require(path.resolve(`${process.env.SCRIPTS_PATH}/index.js`))
+            try {
+              const owners = JSON.parse(process.env.REGISTRY_OWNERS).owners
+              const authorized = await scripts.isAuthorized({github, context}, owners)
+              if (!authorized) {
+                console.error('user is NOT authorized to register buildpack')
+                process.exit(1)
+              }
+            } catch (error) {
+              console.error(error)
+              process.exit(1)
+            }
+
+      - uses: actions/github-script@v2
+        name: Update Registry Index
+        with:
+          github-token: ${{ secrets.BOT_TOKEN }}
+          script: |
+            const path = require('path')
+            const scripts = require(path.resolve(`${process.env.SCRIPTS_PATH}/index.js`))
+            let registryOwners
+            try {
+              await scripts.indexRegistryForBuildpack(
+                {github, context},
+                JSON.parse(process.env.BUILDPACK),
+                process.env.GITHUB_OWNER,
+                process.env.INDEX_REPO
+              )
+            } catch (error) {
+              console.error(error)
+              process.exit(1)
+            }
+
+      - uses: actions/github-script@v2
+        name: Close Issue on Success
+        with:
+          github-token: ${{ secrets.BOT_TOKEN }}
+          script: |
+            const path = require('path')
+            const scripts = require(path.resolve(`${process.env.SCRIPTS_PATH}/index.js`))
+            try {
+              await scripts.closeIssue(
+                {github, context},
+                process.env.GITHUB_OWNER,
+                process.env.INDEX_REPO,
+                'succeeded',
+                '008000'
+              )
+            }
+            catch (error) {
+              console.error(error)
+              process.exit(1)
+            }
+
+      - uses: actions/github-script@v2
+        name: Close Issue on Failure
+        if: ${{ failure() }}
+        with:
+          github-token: ${{ secrets.BOT_TOKEN }}
+          script: |
+            const path = require('path')
+            const scripts = require(path.resolve(`${process.env.SCRIPTS_PATH}/index.js`))
+            try {
+              await scripts.closeIssue(
+                {github, context},
+                process.env.GITHUB_OWNER,
+                process.env.INDEX_REPO,
+                'failed',
```

Comment:
I feel like we should be giving them more information about why an issue "failed". Is that being added somewhere and I'm just missing it?

Reply:
@jromero Currently, we're providing the failure/error details via `stderr`.  So if we need to inspect the details of the failure, it should be in the failed action job/step.

Reply:
A few thoughts:

1. I like the idea of a more helpful message.
2. I don't think we should be suggesting to reopen the issue. I think we should gear them towards reopening a new issue for simplicity and cleanliness purposes.
3. We don't have a tag for this new team yet. TBD.

With that, a proposed error message:


```
Failed to automatically register the buildpack. For more details, see here: <link to the failed job>

If you feel this was in error, you may try again by creating a new issue. If the problem persists or need assistance, you may comment on this issue or contact the maintainers via email at `cncf-buildpacks-maintainers@lists.cncf.io`.
```

Reply:
@jromero  and @dfreilich new example from ^
https://github.com/elbandito/buildpack-registry/issues/183

Reply:
nit: I like empty lines between each sentence but that may just be me.

Reply:
https://github.com/elbandito/buildpack-registry/issues/186